util/mvmap/mvmap.go

```diff
@@ -157,6 +157,11 @@ func (m *MVMap) Get(key []byte) [][]byte {
 		}
 		values = append(values, val)
 	}
+	// Keep the order of input.
```

Comment:
Getting one value takes about 400ns, swap a four values array takes about 4ns.
So it adds about 1% operation time for `Get` function.